tests/python/gpu/test_operator_gpu.py

```diff
@@ -1657,6 +1661,17 @@ def check_rnn_layer_w_rand_inputs(layer):
     for g, c in zip(gs, cs):
         assert_almost_equal(g.asnumpy(), c.asnumpy(), rtol=1e-2, atol=1e-6)
 
+@with_seed()
+@assert_raises_cudnn_disabled()
+def test_rnn_layer():
```

Comment:
Maybe you have not properly rebased? Seems like your code is containing changes from other people's PRs.